src/main/java/com/amazon/ion/impl/bin/IonManagedWriter_1_1.kt

```diff
@@ -299,7 +359,27 @@ internal class IonManagedWriter_1_1(
     }
 
     /**
-     * Writes the `(symbol_table ...)` clause into the encoding expression.
+     * Writes an encoding directive for the current encoding context using the verbose `$ion_encoding::(...)` syntax,
+     * and updates internal state accordingly. This always appends to the current encoding context. If there is nothing
+     * to append, calling this function is a no-op.
+     */
+    private fun writeVerboseEncodingDirective() {
```

Comment:
The "Verbose" methods added in this PR are not new; they were replaced with the equivalent system macro invocations in #987. I've added them back because we'll use them in the macro-aware transcoding when the source stream contains verbose encoding directives.